Smaller improvements to DE about/before/after

sebastianmika opened the pull request:
* In DE `frühestens` and `spätestens` act implicitly as `nach` and `vor` (after and before) on times and may also appear after the time

* The rule `ruleTimeofdayTimeofdayInterval` does match `9Uhr-10` but not the
way more common expression `9-10Uhr`; added the same rule with the
second time as non-latent; actually I am not sure whether the original
rule makes sense at all

* Simple extension of `intersect by ,` to THE formal way in DE to express
a date (i.e. `Freitag, der 13.03.2013`)

General remark: I used UTF-8 characters albeit I saw that the other rules and examples use escaped hex encoding for e.g. German umlaute. If there is any reason to do that (it is not very readable), I will of course change that.

patapizza commented:
@sebastianmika We found that UTF-8 characters didn't work properly so we escaped them. Let's use hex encoding for consistency until we find a solution and update them all.

sebastianmika commented:
@patapizza I replaced the non ASCII chars in the rules I added; the merge conflict is probably something you will resolve, right? At least I wouldn't see immediately how I would do that from my side.

patapizza commented:
@sebastianmika Thanks!
